app/helpers/people_helper.rb

```diff
@@ -32,7 +32,7 @@ def person_link(person, opts={})
   end
 
   def person_image_tag(person, size = :thumb_small)
-    image_tag(person.profile.image_url(size), :alt => person.name, :class => 'avatar', :title => person.name, 'data-person_id' => person.id)
+    image_tag(person.profile.image_url(size), :alt => person.name, :class => 'avatar small', :title => person.name, 'data-person_id' => person.id)
```

Comment:
We use `person_image_link` with different sizes so always adding `small` doesn't sound sensible.